tests/components/amberelectric/conftest.py

```diff
@@ -1,10 +1,56 @@
 """Provide common Amber fixtures."""
 
-from collections.abc import Generator
-from unittest.mock import AsyncMock, patch
+from collections.abc import AsyncGenerator, Generator
+from datetime import date
+from unittest.mock import AsyncMock, Mock, patch
 
+from amberelectric.models.channel import Channel, ChannelType
+from amberelectric.models.site import Site
+from amberelectric.models.site_status import SiteStatus
 import pytest
 
+from homeassistant.components.amberelectric.const import (
+    CONF_SITE_ID,
+    CONF_SITE_NAME,
+    DOMAIN,
+)
+from homeassistant.const import CONF_API_TOKEN
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from .helpers import (
+    CONTROLLED_LOAD_CHANNEL,
+    FEED_IN_CHANNEL,
+    FORECASTS,
+    GENERAL_AND_CONTROLLED_SITE_ID,
+    GENERAL_AND_FEED_IN_SITE_ID,
+    GENERAL_CHANNEL,
+    GENERAL_ONLY_SITE_ID,
+)
+
+from tests.common import MockConfigEntry
+
+MOCK_API_TOKEN = "psk_0000000000000000"
+
+
+async def create_amber_config_entry(site_id):
+    """Create an Amber config entry."""
+    return MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            CONF_API_TOKEN: "TOKEN",
+            CONF_SITE_NAME: "home",
+            CONF_SITE_ID: site_id,
+        },
+        entry_id=site_id,
```

Comment:
the entry_id or the unique_id?

Reply:
I covered this already. The unique id is the concatenation of the site id and channels type otherwise they clash.

Reply:
Though I do have a question: what is the difference?